src/System Application/App/Azure Storage Services Authorization/src/StorageServiceAuthorization.Codeunit.al

```diff
@@ -131,6 +131,20 @@ codeunit 9062 "Storage Service Authorization"
         exit(StorServAuthImpl.ReadySAS(SASToken));
     end;
 
+    /// <summary>
+    /// Uses a pre-generated account SAS (Shared Access Signature) for authorizing HTTP request to Azure Storage Services.
+    /// see: https://go.microsoft.com/fwlink/?linkid=2210398
+    /// </summary>
+    /// <param name="SASToken">A pre-generated SAS token.</param>
+    /// <returns>An account SAS authorization.</returns>
+    [NonDebuggable]
+    procedure UseReadySAS(SASToken: SecretText): Interface "Storage Service Authorization"
```

Comment:
Obsolete the old method, `UseReadySAS(SASToken: Text)`
I can't put these comments on that line 127, since GitHub doesn't allow comments on unmodified lines..

Reply:
`[Obsolete('Reason...', '25.0')]`
and wrap that old method in `#if not CLEAN25` and `#endif`, otherwise the `CLEAN` builds will fail.